### DISCUSSION TITLE
my 54129 pump has 6 wires instead of 4.

### DISCUSSION THREAD

**Discussion by Nightwolve** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#319] [Q&A]
When i connect the PCB in 6 wire configuration there seems to be no power coming to the board. Assuming a short somewhere.
Measured H1 5V and GND and there was no voltage there. When I measure on the 6 wires coming from the pump where 5V and GND should be connected to the PCB I measured 5V without PCB connected so I think it's correct? 
Is it known that 54129 models can have 6 wires?

**Comment by visualapproach** [comment]
I didn't know that but 54123 comes in both flavors so I'm not surprised. Check out where 5V and GND are with a multimeter first and see if it can power the esp.

**Reply by Nightwolve** [reply]
I measured it, perhaps it just cannot power the esp then? Was thinking about opening it up and looking on the motherboard since it should be printed on the silkscreen there to be sure. I didn't solder on any resistors, should I?

**Reply by visualapproach** [reply]
I don't know anything about your model. I can just guess, and I would guess it's the same as 54123 6w version, perhaps with/without some features like hydrojets. Keep the display not connected and measure voltage over 5V and GND when ESP is connected to the pump (CIO). If it's way lower than 5V there is probably a short or insufficient power. How do you determine that the ESP isn't powered?

**Reply by Nightwolve** [reply]
I measured the incoming voltage on the PCB and on the ESP itself. Measured near to 0V so was assuming a short. Guess I'll have to open it up to see which cables are what coming out of the motherboard. Does the 54123 require different pinout?

**Reply by visualapproach** [reply]
Okay good, but you should be able to determine which pins are power and gnd by measuring with nothing connected. But check the motherboard it won't hurt (if you unplug mains)
No idea what the pinout is. Best to assume we don't know, but figure out the power pins first, then after that works we can investigate further.

**Reply by Nightwolve** [reply]
I'll check the motherboard since I measured 5V on 3 different pins (1+4/1+5/1+6). The ones that should be 5V and ground are -5V (polarity was correct). So to avoid confusion I'll just check motherboard.

**Reply by Nightwolve** [reply]
I checked, I measure -4,5V over 2 channels(polarity om multimeter is correct). Taking your pinout at reference I have on the following wires 4,5V. The GND matches your schematic. I measure -4,5V between 1 and 2(schematics vcc and gnd) and also -4,6V between 2 and 5. Is the voltage enough to start up the ESP?

On the motherboard itself there is no silkscreen with the connector layout.
![IMG_20220702_182949](https://user-images.githubusercontent.com/36737388/177009088-31d31645-c4e9-4596-a2e7-2124c58170e9.jpg)

When I plug in the PCB I measure 426mV on the ESP and the 221

**Reply by visualapproach** [reply]
Just to be clear: Red probe on pin1 (rightmost on your pic) and black probe on pin 2 (dotted wire in your pic). You should read +5V. Not minus. If you read -5V (-4.5V) they have switch the order. You must connect +5V to pin1 on the PCB.

**Reply by Nightwolve** [reply]
I switched the cable connections 180° and it works now! Only struggling with the Wifi connection since I cannot login anymore on the web portal, I assume the signal just isn't strong enough since it worked at my desk before.
